src/core/MRMedia.js

```diff
@@ -0,0 +1,242 @@
+import * as THREE from 'three';
+
+import { MRDivEntity } from 'mrjs/core/MRDivEntity';
+
+import { mrjsUtils } from 'mrjs';
+
+/**
+ * @class MRMedia
+ * @classdesc Base html media entity represented in 3D space. `mr-media`
+ * @augments MRDivEntity
+ */
+export class MRMedia extends MRDivEntity {
+    /**
+     * @class
+     * @description Constructs a base media entity using a UIPlane and other 3D elements as necessary.
+     */
+    constructor() {
+        super();
+        this.attachShadow({ mode: 'open' });
+
+        // Create the object3D. Dont need default value for geometry
+        // until the connected call since this will get overwritten anyways.
+        let material = new THREE.MeshStandardMaterial({
```

Comment:
I would like to always use `const` specification instead of `let` for immutable variables to avoid unexpected reassignment.

Reply:
√